src/pages/authentication/connect.md

```diff
@@ -1,187 +0,0 @@
----
-title: Stacks Connect
-description: Learn what Connect is and how to integrate it into an app.
-experience: beginners
-duration: 15 minutes
-images:
-  large: /images/pages/connect.svg
-  sm: /images/pages/connect-sm.svg
----
-
-## Introduction
-
-Stacks Connect is a JavaScript library for integrating Stacks authentication, data storage, and smart contracts into your app.
-
-The library empowers you to:
-
-- Register new users with a pre-built onboarding flow that quickly educates them as to the privacy benefits of using your app with Stacks authentication and provisions a "Secret Key" that secures their identity and data against the Stacks blockchain.
-- Authenticate users when they return to your app using that same Secret Key.
-- Prompt users to sign transactions with smart contracts as written in Clarity and published to the Stacks blockchain.
-
-## How does this compare to `stacks.js`?
-
-Connect uses stacks.js under the hood to construct and decode authentication requests. It provides pre-built onboarding UI that educates users as to how your app is more secure for having implemented Stacks authentication. You will need to use `stack.js` packages for storage and transactions.
-
-## Start building with Stacks Connect
-
-Head over to the [to-do app tutorial](/authentication/building-todo-app) to learn how to build apps with Stacks Connect. For interaction with Stacks accounts and smart contracts with Stacks Connect see the [transaction sigining section](/smart-contracts/signing-transactions).
-
-## Installation
-
-With yarn:
-
-```bash
-yarn add @stacks/connect
-```
-
-With npm:
-
-```bash
-npm install --save @stacks/connect
-```
-
-## Usage
-
-### AuthOptions
```

Comment:
I think this chapter would be nice in some reference - maybe of the auth package?

Reply:
We have the typedoc generated reference, but that's probably not the right place for this. Perhaps in addition to the todo tutorial we need reference guide pages.